schema/url/pass/plugin-pieplot.raml

```diff
@@ -18,12 +18,19 @@ types:
         required: false
       xAxisVariable: 
         type: VariableSpec
+      # TODO rename to geoAggregateVariable after b57
       facetVariable: 
         type: VariableSpec[]
         maxItems: 2
         required: false
       valueSpec: 
         type: ValueSpec
+      longitudeVariable?:
+        type: VariableSpec
+      latitudeVariable?:
+        type: VariableSpec
+      viewport?:
+        type: GeolocationViewport
 
   PieplotConfig:
```

Comment:
looks like plot.data is returning a viewport as part of the pieplot$config. Should be added here as well?